README.md

```diff
@@ -61,7 +61,7 @@ $ mkdir $GOPATH/src/kubevirt.io && cd $GOPATH/src/kubevirt.io
 $ git clone https://github.com/kubevirt/containerized-data-importer && cd containerized-data-importer
 $ make cluster-up
 $ make cluster-sync
-$ ./cluster/kubect.sh .....
+$ ./cluster-up/kubectl.sh .....
```

Comment:
Might want to create a section for this and point out something like "The development tools include a version of kubectl that you can use to communicate with the cluster.  The wrapper script to communicate with your CI cluster can be invoked using `./cluster-up/kubectl.sh`.

Probably not worth going through and describing how to add your own kubectl; if people want to go that route they can figure it out.